resolver/resolvers.go

```diff
@@ -90,6 +90,16 @@ func createResolver(resolverURL, source string) (*Resolver, bool, error) {
 		return nil, false, fmt.Errorf("invalid resolver IP")
 	}
 
+	// Add default port for scheme if it is missing.
+	if u.Port() == "" {
+		switch u.Scheme {
+		case ServerTypeDNS, ServerTypeTCP:
+			u.Host += ":53"
+		case ServerTypeDoT:
+			u.Host += ":853"
+		}
+	}
+
```

Comment:
I guess we should update the validation regex in the config as well?

Reply:
The validation is very generous anyway and already includes this change. Unless we want to improve it?

Reply:
You're right the validation regex is already lax enough. Though, we should update the help text: https://github.com/safing/portmaster/blob/develop/resolver/config.go#L99